docroot/themes/humsci/humsci_basic/src/scss/elements/_details.scss

```diff
@@ -0,0 +1,79 @@
+details {
+  margin: hb-calculate-rems(16px) 0;
+}
+
+summary {
+  padding: hb-calculate-rems(16px);
+  position: relative;
+  font-weight: hb-theme-font-weight('semibold');
+  cursor: pointer;
+
+  @include hb-colorful {
+    border: $hb-thin-border;
+    @include hb-pairing-color('border-color', 'secondary');
+    @include hb-pairing-color('background-color', 'secondary');
+    @include hb-global-color('color', 'white');
+  }
+
+  // Removes the default arrow icon
+  &::-webkit-details-marker {
+    display: none;
+  }
+
+  &::after {
+    content: '';
+    display: block;
+    height: hb-calculate-rems(20px);
+    width: hb-calculate-rems(20px);
+
+    position: absolute;
+    top: hb-calculate-rems(18px);
+    right: hb-calculate-rems(18px);
+
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+
+    @include hb-colorful {
+      @include hb-menu-plus-icon('white');
+    }
+
+    details[open] & {
+      @include hb-colorful {
+        @include hb-menu-minus-icon('white');
+      }
+    }
+  }
+
+  &:hover,
+  &:focus,
+  details[open] & {
+    @include hb-colorful {
+      @include hb-pairing-color('background-color', 'secondary-darken-12');
+      @include hb-pairing-color('border-color', 'secondary-darken-12');
+    }
+  }
+
+  &:focus {
+    @include hb-colorful {
+      outline: none;
+      @include hb-global-color('border-color', 'white');
```

Comment:
is this white border intentional? looks kinda weird

Reply:
it only applies on the pressdown and then if you close the accordion and keep focus on it

Reply:
Yeah, we want the border that so it matches the focus state that the buttons have it. I did fix it so that the border is also there when the accordion is open and the button has focus.